BMM.Core/ViewModels/ExploreNewestViewModel.cs

```diff
@@ -104,7 +103,7 @@ public override async Task<IEnumerable<IDocumentPO>> LoadItems(CachePolicy polic
             var age = _config.SendAgeToDiscover ? _user.GetUser().Age : null;
             var docs = (await Client.Discover.GetDocuments(age, await _deviceInfo.GetCurrentTheme() , policy)).ToList();
             await _streakObserver.UpdateStreakIfLocalVersionIsNewer(docs);
-            var filteredDocs = await HideElementsInList(HideTeaserPodcastsInList(docs));
```

Comment:
@akrol95 I believe this is no longer needed since we no longer show the podcast twice and rather return different home screens based on app version.